src/installer/installer.go

```diff
@@ -83,9 +83,15 @@ func (i *installer) InstallNode() error {
 	if err != nil {
 		return err
 	}
+
 	i.UpdateHostStatus(WritingImageToDisk)
+	image, err := i.getRchosImage()
```

Comment:
Please move to another package.
The mapping and the helper functions.
you can name it "supportedOpenshiftVersion"